### ISSUE TITLE
Getting same output for every input in tensorflow js model but the keras model from which it is converted is giving right answers.

### ISSUE THREAD

**Issue by Darkshadow9799** [tensorflow/tfjs#3913]
This template is for miscellaneous issues not covered by the other issue categories.

For questions on how to work with TensorFlow.js, or support for problems that are not verified bugs in TensorFlow.js, please go to [StackOverflow](https://stackoverflow.com/tags/tensorflow.js).

**Comment by Darkshadow9799** [comment]
The issue got resolved by normalizing the input image. This can be done by using : tf.scalar(127.55) which will normalize the tensor (or image array) between [-1,1].
